nvflare/private/fed/client/client_executor.py

```diff
@@ -214,6 +183,17 @@ def start_app(
         )
         thread.start()
 
+    def _get_job_launcher(self, job_meta: dict, fl_ctx: FLContext) -> JobLauncherSpec:
+        engine = fl_ctx.get_engine()
+        fl_ctx.set_prop(FLContextKey.JOB_META, job_meta, private=True, sticky=False)
+        engine.fire_event(EventType.GET_JOB_LAUNCHER, fl_ctx)
+
+        job_launcher = fl_ctx.get_prop(FLContextKey.JOB_LAUNCHER)
+        if not (job_launcher or isinstance(job_launcher, list)):
```

Comment:
This isn't right. We expect job_launcher to be a list. The condition should be "and", not "or".

This should simply be:
if not isinstance(job_launcher, list)

Reply:
The logic is correct. it is the same as "not job_launcher and not isinstance(job_launcher, list)". But the "not job_launcher" is not necessary.